packages/cli/commands/project/cloneApp.js

```diff
@@ -0,0 +1,177 @@
+const fs = require('fs');
```

Comment:
Can we add some tests for this command so we can assure that future changes maintain the behavior?

Reply:
Going to implement tests in a separate PR, so I can add tests for `migrate-app` at the same time. Also so that we can hopefully test this command in tomorrow's bug bash.